test/io/ferguson/read-class-reorder.chpl

```diff
@@ -0,0 +1,29 @@
+/*class Base {
+}
+
+class Parent : Base {
+  var x: int;
+  var y: int;
+}
+*/
+
+class Child /*: Parent*/ {
```

Comment:
Interesting.  Is this something we'd expect/like to work down the road?  Might be worth adding a feature request/bug future.  Also, might be worthwhile to check the reordering within either parent or child fields, as opposed to across the boundaries.